packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -58,5 +60,6 @@ StorefrontPreview.propTypes = {
                 `${propName} is a required function for ${componentName} when enabled is true`
             )
         }
-    }
+    },
+    getAdditionalSearchParams: PropTypes.array
```

Comment:
This was changed from `func` to `array`, but the name and usage are still based on it being a function. You're more familiar with the use case, so I'm happy with whichever implementation you think is best, but we should make sure that everything is consistent.

Reply:
I'm not sure what you as asking me to change, to be honest. A link to a wikipedia page isn't helpful. This just needs to be an array. it's documented [here](https://salesforce.quip.com/4jFkAcbUe2HM#temp:C:OdS003008f5510d43d2a7b77cb3e) in the quip as an example of what we expect.

Reply:
The prop type used here is `PropTypes.array`, indicating that the component should be used as `<StorefrontPreview getAdditionalSearchParams={[{name: 'vse', value: '...'}]} />`. But the name _get_ AdditionalSearchParams implies that it's a function, and that's the way that the prop is used in the test file in this PR and in the Runtime Admin PR. (i.e. usage is `<StorefrontPreview getAdditionalSearchParams={() => [...]} />`)

The smallest fix is to change this back to `PropTypes.func`, as that aligns with the current code. However, if the intention here is to change everything to use a static array, rather than a function returning an array, then the rest of the code will also need to be changed. I don't have any opinion on which way should be chosen.

(If you're referring to my link for "bike shed", that's just because I've used the phrase before and found that people didn't know it. But all I meant was that naming the prop is a triviality, so do whatever.)

Reply:
ok, thanks for the clarification @wjhsf 

I've done the following:

1. renamed the prop to `additionalSearchParams`
2. updated it to be a static array based on your original feedback. that was done here, as well as in the runtime-admin-ui PR

let me know if there's anything else needed here.